README.md

```diff
@@ -99,6 +100,65 @@ Then visit http://localhost:8000/.
 Note that the server, `./cg-fake-uaa`, must also be running in order
 for the client to work.
 
+# How to use `example-client` with real cloud.gov identity provider
+
+Although we built `cg-fake-uaa` for testing real client applications with UAA, 
+you can use the client in `example-client` to demonstrate the capability
+of the cloud.gov [identity provider](https://cloud.gov/docs/services/cloud-gov-identity-provider/).
+
+In this example you run the DO-NOT-USE-IN-PRODUCTION `example-client.js` node application 
+to authenticate real users. You'll need a [cloud.gov account](https://account.fr.cloud.gov/signup), 
+and you should already be logged-in (`cf login --sso`).
+
+First we'll `push` the example-client with a random route, and get that route name
+for later use:
+
+```bash
+cd example-client/
+cf push -m 128M --no-start --random-route id-example
+app_route=$(cf apps | grep '^id-example' | awk '{print $NF}')
```

Comment:
Wow, for the past 2 decades I assumed there had to be an `export` in front of this, but apparently not!  Thanks for teaching me something :)

Reply:
`export` needed for child processes, not for current process.  

What is annoying is it that `cf` doesn't have a `--format json` for output. 😞 

Thanks!  And good luck!